theseus/geometry/lie_group.py

```diff
@@ -44,6 +45,17 @@ def _init_data(*args: Any) -> torch.Tensor:
     def dof(self) -> int:
         pass
 
+    @staticmethod
+    @abc.abstractmethod
+    def rand(
+        *size: int,
+        generator: Optional[torch.Generator] = None,
```

Comment:
Can this be `Optional`, what's the behavior when it is left as `None`?

Reply:
It will use the default generator for `torch.rand()`.